.github/workflows/spellcheck.yml

```
@@ -0,0 +1,13 @@
+name: Run Spellcheck
+
+on: [push, pull_request]
+
+jobs:
+  spellcheck:
+    name: Check Spelling
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check spelling
+    - uses: check-spelling/check-spelling@v0.0.19
```

Error: .github#L1
every step must define a `uses` or `run` key

---

See the docs on the action, you need to define patterns and other things. Also, the uses key should be part of the step:
```suggestion
    steps:
    - uses: actions/checkout@v2
    - name: Check spelling
       uses: check-spelling/check-spelling@v0.0.19
```